### PR TITLE
Update deps

### PR DESCRIPTION
**Pull request by johannespfeiffer**
Follow-up PR to update dependencies.

If merged, I suggest that we generate fresh schema files, as they differ quite a lot.

### CODE REVIEW

**Review by lera — APPROVED** [review]

File: lib/utils/index.ts [hunk 1]
```diff
@@ -1,2 +1,7 @@
-export * from "./authorizeSilently";
+import { authorizeSilently } from "./authorizeSilently";
+
 export * from "./serializeGraphQLError";
+
+export const utils = {
+    authorizeSilently,
+}
```
Comment (sinahashemy) [inline lib/utils/index.ts:7]: 🛠 isn't need for one newline here?

File: package.json [hunk 2]
```diff
@@ -15,39 +15,39 @@
...
+    "tslint": "^6.1.3",
+    "typescript": "^4.5.5",
+    "webpack": "^4.46.0",
+    "webpack-cli": "^4.9.2"
   },
   "bin": {
     "kontist": "./cli/index.js"
   }
-}
+}
```
Comment (sinahashemy) [inline package.json:53]: 🛠 isn't need for one newline here?

File: tests/graphql/account.spec.ts [hunk 3]
```diff
@@ -1,9 +1,9 @@
-import { expect } from "chai";
 import * as sinon from "sinon";
 
+import { Account } from "../../lib/graphql/account";
 import { Client } from "../../lib";
 import { KontistSDKError } from "../../lib/errors";
-import { Account } from "../../lib/graphql/account";
+import { expect } from "chai";
```
Comment (sinahashemy) [inline tests/graphql/account.spec.ts:6]: 🛠 in other repositories we kept the external library at top of the file, then the internal libraries!

File: tests/graphql/user.spec.ts [hunk 4]
```diff
@@ -1,9 +1,9 @@
-import { expect } from "chai";
 import * as sinon from "sinon";
 
 import { Client } from "../../lib";
 import { KontistSDKError } from "../../lib/errors";
 import { User } from "../../lib/graphql/user";
+import { expect } from "chai";
```
Comment (sinahashemy) [inline tests/graphql/user.spec.ts:6]: 💭 like the previous comment.

**Review by sinahashemy — APPROVED** [review]
Thank you for updating the libraries 👍. I leave some comments.

**Comment by johannespfeiffer** [comment]
@sinahashemy valid point with the style changes. We do use `tslint` in this project and just mostly stick to the latest defaults via `tslint:latest`. Probably that is where it came from.

**Pull request merged**